doc/whats_new/v0.22.rst

```diff
@@ -85,6 +85,11 @@ Changelog
   :func:`datasets.fetch_20newsgroups` and :func:`datasets.fetch_olivetti_faces`
   . :pr:`14259` by :user:`Sourav Singh <souravsingh>`.
 
+- |Enhancement| `make_classification` in :func:`datasets.samples_generator` now 
+  accepts array-like `weights` parameter, i.e. list or numpy.array, instead of 
+  list only.
+  :pr:`14764` by :user:`Cat Chenal <CatChenal>`, with thanks to `WiMLDS <WiMLDS>`.
```

Comment:
```suggestion
  :pr:`14764` by :user:`Cat Chenal <CatChenal>`, with thanks to *WiMLDS*.
```

Reply:
Certainly not. That would downgrade @WiMLDS's contributions to Opensource & Scikit-learn [in particular](https://github.com/WiMLDS/nyc-2019-scikit-sprint).

Reply:
This would not link to anything and will consider “WIMDS” a codeblock

Reply:
It may work with a :user: tag tho

Reply:
Something like should link to the @WiMLDS's account

```
:user:`WiMLDS <WiMLDS>
```

Reply:
In other words, this would link you and @WiMLDS

```
- |Enhancement| :func:`datasets.make_classification` now accepts array-like
  `weights` parameter, i.e. list or numpy.array, instead of list only.
  :pr:`14764` by :user:`Cat Chenal <CatChenal>`, with thanks to
  :user:`WiMLDS <WiMLDS>`.
```

Reply:
> Certainly not. That would downgrade @WiMLDS's contributions to Opensource & Scikit-learn in particular.

Sorry, I misunderstood the intend of the string. I initially thought you were trying to make it bold or italic to denote an organization. I can see now you were trying to link to the organization on github. The above snippet should correctly link to their organization.

Reply:
I'm not sure about this, we usually acknowledge individuals not organizations in release notes. For funding organizations are typically mentioned in https://scikit-learn.org/stable/about.html#funding). I think we could maybe add a section for WiMLDS and similar partner organizations there? The problem with acknowledgements of organizations in release notes is that most contributions have some sort of organization behind it ( Numfocus, conference where the sprint happened, company who allowed its employee to contribute during work time, etc), and then deciding to acknowledge some but not others is tricky.

Reply:
Thanks for the information. The mention is then out of place in the release notes. I will remove it & add #WiMLDS in the final commit.